src/Native/Runtime/unix/unixasmoffsets.cpp

```diff
@@ -0,0 +1,19 @@
+//
+// Copyright (c) Microsoft Corporation.  All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+#ifdef _ARM_
+
+#define PLAT_ASM_OFFSET(offset, cls, member) OFFSETOF__##cls##__##member 0x##offset##h
```

Comment:
This is strange - using both the "0x" prefix and "h" suffix. I'd use just the "0x" prefix, the assembler accepts that. It would not be a good idea to do it the other way (using just the "h") though. The assembler has a problem with constants ending with "h" where it sometimes strips the "h" when passing the constant as a macro parameter. I had a lot of fun with these at the beginning of CoreCLR port.